domain/src/main/java/com/bottlerocketstudios/brarchitecture/domain/models/Project.kt

```diff
@@ -0,0 +1,9 @@
+package com.bottlerocketstudios.brarchitecture.domain.models
+
+import java.time.ZonedDateTime
+
+data class Project(
+    val name: String,
+    val key: String,
+    val updatedOn: ZonedDateTime?,
+)
```

Comment:
Hey @LuisRocketRangel just in case for this domain model object to help differentiate between other model objects let's extend that DomainModel interface. Would look something like this: 

```
data class Project(
    val name: String,
    val key: String,
    val updatedOn: ZonedDateTime?,
) : DomainModel
```